autogpt/prompts/prompt.py

```diff
@@ -46,13 +46,304 @@ def build_default_prompt_generator() -> PromptGenerator:
     return prompt_generator
 
 
-def construct_main_ai_config() -> AIConfig:
+def construct_main_ai_config(config_file: str = None) -> AIConfig:
```

Comment:
This is an extremely long function. Would it be possible to break it up? e.g. the functionality for creating, changing, and deleting configs could be separated cleanly.

Reply:
Thanks and absolutely right, I was already busy with it. Can you hook up on discord, I would like to hear your opinion, maybe in the end it's even better to close this one and re-factor on a new PR.